api/models/tiers.go

```diff
@@ -31,6 +31,15 @@ var MemberLevelFromAmount = map[int64]MemberLevel{
 	50: Premium,
 }
 
+// MemberLeveltoAmount convert MemberLevel to amount
+var MemberLevelToAmount = map[MemberLevel]int64{
```

Comment:
Hrmm, this is basically `MemberLevelFromAmount` but the keys and values are just switched. Can you think of another way so that we don't have to do this.